### PR TITLE
Added test for sqlite automatic numeric conversion

### PR DESCRIPTION
**Pull request by Pchelolo**
Added a test for a problem described in https://github.com/wikimedia/restbase-mod-table-sqlite/pull/8

### CODE REVIEW

File: test/functional/types.js [hunk 1]
```diff
@@ -198,6 +199,40 @@ describe('Types', function() {
...
+            return router.request({
+                uri: '/restbase.cassandra.test.local/sys/table/typeTable/',
+                method: 'put',
+                body: {
+                    table: "typeTable",
+                    attributes: {
+                        string: 'num_string_test',
+                        int: 10,
+                        boolean: false,
+                        num_string: '2015'
```
Comment (d00rman) [inline test/functional/types.js]: Would be better to set it to `'0'` so that we also test the corner case for `if(!value)`

Comment (Pchelolo) [inline test/functional/types.js]: @d00rman Indeed. Updated the PR
